src/models/authentication.js

```diff
@@ -2,6 +2,7 @@ import Model from './model'
 
 class authentication extends Model({
   accessToken: undefined,
+  error: undefined,
```

Comment:
is there a need to have this error here? As you're using Jungle's boilerplate, it should handle all the errors on the error reducer